src/App.js

```diff
@@ -2,18 +2,10 @@ import React from 'react';
 import './App.css';
 import Calculator from './component/Calculator.js';
 
-class App extends React.Component {
-  // eslint-disable-next-line no-useless-constructor
-  constructor(props) {
-    super(props);
-  }
-
-  // eslint-disable-next-line class-methods-use-this
-  render() {
-    return (
+function App() {
```

Comment:
- [ ] Kindly use `arrow functions` instead of the `function` keyword, using ES6 syntax is part of your learning of best practices. 😄 
    - Why use arrow functions almost everywhere? 🚀 

        - Scope safety: When arrow functions are used consistently, everything is guaranteed to use the same thisObject as the root. If even a single standard function callback is mixed in with a bunch of arrow functions there's a chance the scope will become messed up.
        - Compactness: Arrow functions are easier to read and write. (This may seem opinionated so I will give a few examples further on.)
        - Clarity: When almost everything is an arrow function, any regular function immediately sticks out for defining the scope. A developer can always look up the next-higher function statement to see what the thisObject is.

Reply:
This issue is well addressed. Nice job! :pray: